plugins/modules/dcnm_service_route_peering.py

```diff
@@ -3506,7 +4321,9 @@ def main():
         debug=dict(required=False, type="bool", default=False),
     )
 
-    module = AnsibleModule(argument_spec=element_spec, supports_check_mode=True)
+    module = AnsibleModule(
+        argument_spec=element_spec, supports_check_mode=True
```

Comment:
I think we discussed this a few weeks back but looks like `check_mode` is still specified as a module argument.  This is an ansible infra level arg so we need to update the this module and `service_node` and `service_policy` modules if they have the same issue.

@praveenramoorthy recently made a similar update/fix for dcnm_vrf and dcnm_network modules

See: https://github.com/CiscoDevNet/ansible-dcnm/pull/88/files